build.sh

```diff
@@ -83,7 +83,7 @@ concurrency=$(node -p 'Math.max(1, require("os").cpus().length - 1)')
 
 echo "============================================================================================="
 echo "building..."
-time lerna run $bail --stream --concurrency=$concurrency $runtarget || fail
+time npx lerna run $bail --stream $runtarget || fail
```

Comment:
Add back the concurrency flag (which was added after the original authoring of this PR):
```suggestion
time npx lerna run $bail --stream --concurrency=$concurrency $runtarget || fail
```

Thanks @TheRealAmazonKendra for catching that.